irohad/main/raw_block_loader.hpp

```diff
@@ -35,9 +35,9 @@ namespace iroha {
      * This class will be useful for creating test environment
      * and testing pipeline.
      */
-    class BlockInserter {
+    class BlockLoader {
      public:
-      explicit BlockInserter(std::shared_ptr<ametsuchi::MutableFactory> factory);
+      BlockLoader() = default;
```

Comment:
Logger has to be initialized in constructor, `std::shared_ptr<ametsuchi::MutableFactory> factory_;` is not required now.